deployments/helm/honeydipper/values.yaml

```diff
@@ -60,6 +60,7 @@ drivers:
     service:
       type: LoadBalancer
       port: 8080
+      nodePort: 0
```

Comment:
0 will evaluate to false in yaml, right? Would just having `false` be more clear?

Reply:
The `type` has to be consistent in helm. If we use boolean here, we won't be able to specify numerical value to override it.